README.md

```diff
@@ -3,54 +3,134 @@
 [![Build Status](https://travis-ci.org/redhataccess/pantheon.png)](https://travis-ci.org/redhataccess/pantheon)
 [![Code Coverage](https://codecov.io/gh/redhataccess/pantheon/branch/master/graph/badge.svg)](https://codecov.io/github/redhataccess/pantheon?branch=master)
 
-Pantheon 2 is a modular documentation management and publication system based on asciidoc
-and built on top of Apache sling.
+Pantheon 2 is a modular documentation management and publication system based on Asciidoc
+and built on top of Apache Sling.
 
-### Prerequsistes
-Podman
-Buildah
-Java
+* [Contributing to Pantheon](#contributing-to-pantheon)
+* [Installing Pantheon](#installing-pantheon)
+ * [Prerequisites](#prerequisites)
+ * [Building the application](#building-the-application)
+ * [Unit tests](#unit-tests)
+ * [Running the application](#running-the-application)
+ * [Live deploy of code](#live-deploy-of-code)
+ * [Using the application](#using-the-application)
+ * [Other use cases](#other-use-cases)
+ * [Developing the frontend code](#developing-the-frontend-code)
+* [End user documentation](#end-user-documentation)
 
-### Build the application
-_(All commands from here on will be assumed to be ran from the project's root directory)_
+## Contributing to Pantheon
+
+### Prerequisites
+
+* You must have an account on GitHub.  
+  [Signing up for a new GitHub account](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account)
```

Comment:
This link feels awkward just sitting here. Could we introduce the concept ("for more information, see") or make the entire prerequisite a link to the help article covering the topic?

Reply:
> or make the entire prerequisite a link to the help article covering the topic

@robin-owen, isn't that what we have here? The text of the prereq is a link to the help article.